libsolidity/analysis/FunctionCallGraph.cpp

```diff
@@ -0,0 +1,109 @@
+#include <libsolidity/analysis/FunctionCallGraph.h>
+//#include <libsolutil/CommonData.h>
+
+#include <boost/range/adaptor/reversed.hpp>
+
+using namespace solidity::frontend;
+
+void FunctionCallGraphBuilder::analyze()
+{
+	for (auto const* contract: m_currentContract->annotation().linearizedBaseContracts  | boost::adaptors::reversed)
+		if (contract->constructor())
+			contract->constructor()->accept(*this);
+
+	for (VariableDeclaration const* stateVar: m_currentContract->stateVariablesIncludingInherited())
+		if (stateVar->value())
+		{
+			m_currentNode = stateVar;
+			stateVar->value()->accept(*this);
+			m_mapping.insert(std::pair(stateVar, std::move(m_currentMapping)));
```

Comment:
```suggestion
			m_mapping.emplace(stateVar, std::move(m_currentMapping));
```
not too important, but...